bin/po2android.js

```diff
@@ -117,10 +120,10 @@ ${ indent }</string-array>
 
 if ( require.main === module ) {
 	if ( process.stdin.isTTY ) {
-		const potFileName = process.argv[ 2 ];
-		const destination = process.argv[ 3 ];
-		const potFileContent = fs.readFileSync( potFileName );
-		const xmlOutput = po2Android( potFileContent, process.argv[ 3 ] );
+		const destination = process.argv[ 2 ];
+		const potFiles = process.argv.slice( 3 );
+		const potFilesContent = potFiles.map( (file) => fs.readFileSync( file ) );
+		const xmlOutput = po2Android( potFilesContent );
```

Comment:
Now the `po2android` and `po2ios` scripts support multiple POT files, they basically merge all strings into a single object that is later on used for generating the localization strings files.